docs/source/usage/param/particles/init.rst

```diff
@@ -1,7 +1,7 @@
 .. _usage-particles:
```

Comment:
So should the link then also be `.. usage-params-particles:` to reflect the tree structure?

Reply:
yep, done now (`usage-params-core-...`)